### PR TITLE
Add localstack to test state locally

### PR DESCRIPTION
**Pull request by hemslo**
Add localstack to test state locally.

Upgrade `boto3` version so it can point to localstack via env `AWS_ENDPOINT_URL`.

To test it locally:

```bash
# start
$ make localstack

# list all s3 buckets
$ aws --endpoint-url=http://localhost:4566 s3 ls

# set all envs for qontract-cli to work with localstack
$ export $(cat dev/localstack/.env.local | xargs)

# set state
$ qontract-cli --config config.toml state set terraform-resources k v

# get state
$ qontract-cli --config config.toml state get terraform-resources k

# verify s3 objects
$ aws --endpoint-url=http://localhost:4566 s3 ls s3://app-interface-state --recursive
```

[APPSRE-8725](https://issues.redhat.com/browse/APPSRE-8725)

### CODE REVIEW

File: tools/qontract_cli.py [hunk 1]
```diff
@@ -2347,7 +2347,7 @@ def cluster_admin(ctx, org_name, cluster_name):
 
 
 @root.group()
-@environ(["APP_INTERFACE_STATE_BUCKET", "APP_INTERFACE_STATE_BUCKET_ACCOUNT"])
```
Comment (patjlm) [inline tools/qontract_cli.py:2350]: account was not used?

**Comment by patjlm** [comment]
Looks nice! LGTM (i'd like someone else to review as well ;-) )

**Review by chassing — APPROVED** [review]
Nice!! LGTM

**Comment by chassing** [comment]
I'll add localstack support into [qontract-development-cli](https://github.com/app-sre/qontract-development-cli) too - https://issues.redhat.com/browse/APPSRE-8749

**Review by cubismod — APPROVED** [review]

**Comment by chassing** [comment]
For referene: https://github.com/app-sre/qontract-development-cli/pull/39

**Pull request merged**